### PR TITLE
Update yml

### PR DESCRIPTION
**Pull request by adebiyial**
The `yml` file has been updated to run with yarn

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ node_modules
 nuxt.d.ts
 .output
 .env
-.DS_Store
\ No newline at end of file
+.DS_Store# Layer0 generated build directory
```
Comment (justinmetros) [inline .gitignore:7]: Please correct comment

File: package.json [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
     "build": "nuxi build",
     "start": "node .output/server/index.mjs",
     "get-shopify-fragment-types": "node ./apollo/schemaQuery.mjs",
     "deploy": "layer0 deploy"
   },
+  "name": "nuxtshop",
+  "private": true,
+  "license": "MIT",
+  "engines": {
+    "node": "^14.16.0 || ^16.11.0 || ^17.0.0"
```
Comment (justinmetros) [inline package.json:16]: @adebiyial Does layer0 work with node16 yet? If not we should add something to check

File: package.json [hunk 3]
```diff
@@ -1,20 +1,28 @@
...
+    "layer0:build": "layer0 build",
+    "layer0:deploy": "layer0 deploy",
     "dev": "nuxi dev",
     "build": "nuxi build",
     "start": "node .output/server/index.mjs",
     "get-shopify-fragment-types": "node ./apollo/schemaQuery.mjs",
     "deploy": "layer0 deploy"
   },
+  "name": "nuxtshop",
+  "private": true,
```
Comment (justinmetros) [inline package.json:13]: We might as well remove this now

**Review by justinmetros — CHANGES_REQUESTED** [review]
Few small comments but otherwise the layer0 stuff looks good

File: package.json (hunk 3 above)
Comment (justinmetros) [reply package.json:13]: the `private` i mean

Comment (adebiyial) [reply package.json:13]: I left the name.

File: package.json (hunk 2 above)
Comment (adebiyial) [reply package.json:16]: Updated.

File: package.json (hunk 3 above)
Comment (adebiyial) [reply package.json]: Done.

**Review by justinmetros — APPROVED** [review]
👍

**Pull request merged**